instrumentation/rocketmq/src/main/java/brave/rocketmq/clients/TraceConstants.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2013-2023 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.rocketmq.clients;
+
+/**
+ * @author JoeKerouac
+ * @date 2023-01-10 15:19
+ */
+public class TraceConstants {
```

Comment:
look at how other packages define constants and use similar naming scheme

Reply:
I've been busy lately. I will deal with it in the next two days.